app/initializers/component-router-injector.js

```diff
@@ -0,0 +1,8 @@
+export function initialize(application) {
```

Comment:
@ritikamotwani thank you for the PR. Can you please move the action definition to the controller. This won't be required then to navigate between routes :)